### DISCUSSION TITLE
Input Text do Formulário Bricks Incorreto

### DISCUSSION THREAD

**Discussion by stefanobarcellos** [mercadopago/sdk-js#215] [General]
O input que a pessoa deve escrever o e-mail está como type="text" quando deveria ser type="email"

Isso gera problemas com os corretores ortográficos dos sistemas Android e IOS, pois eles colocam a primeira letra do e-mail em maiúsculo, acarretando induzimento a erro por parte do cliente. Para arrumar isso por enquanto eu fiz um JS que fica observando e muda o type.

====================

	// Função para ajustar o input de e-mail
	function adjustEmailInput() {
		const input = document.querySelector('input[name="EMAIL"]');
		
		if (input) {
			input.type = 'email';
			input.autocapitalize = 'none';
		}
	}

	// Função para observar mudanças no DOM
	function observeDOMChanges() {
		const observer = new MutationObserver((mutationsList) => {
			for (const mutation of mutationsList) {
				if (mutation.type === 'childList') {
					// Verifica se o input com name="EMAIL" foi adicionado
					const emailInput = document.querySelector('input[name="EMAIL"]');
					if (emailInput) {
						adjustEmailInput();
						// Opcional: desconectar o observador se não precisar mais observar
						// observer.disconnect();
						break;
					}
				}
			}
		});

		// Configura o observer para observar o corpo do documento
		observer.observe(document.body, { childList: true, subtree: true });
	}

	// Chama a função para iniciar a observação
	observeDOMChanges();
	
====================

Obrigado.

**Comment by meliguicarvalho** [comment]
Boa tarde Stéfano. 

Obrigado por entrar em contato.

Vamos analisar esse caso que você reportou, e entramos em contato quando houver uma solução.

**Comment by meliguicarvalho** [comment]
Bom dia Stéfano.

Obrigado por encontrar essa inconsistência.

Realizamos um ajuste, e agora já colocamos em produção uma nova versão do Brick com a correção do tipo de campo.
